lib/ExpensiMark.js

```diff
@@ -250,4 +254,100 @@ export default class ExpensiMark {
 
         return replacedText;
     }
+
+    /**
+     * Modify text for Quotes replacing chevrons with html elements
+     *
+     * @param {RegExp} regex
+     * @param {String} textToCheck
+     *
+     * @returns {String}
+     */
+
+    modifyTextForQuote(regex, textToCheck) {
+        console.log('tetxttocheck =', textToCheck);
+        let replacedText = '';
+        let TextToFormat = '';
+        let isCodeFence;
+        const codefenceSplitted = [];
+        const match = textToCheck.match(regex);
+        const match_codefence = textToCheck.match(/<pre>(\s|\S)+<\/pre>/gm);
+
+        // if there's matches we need to modify the quotes
+        if (match !== null) {
+            // split all the codefences in lines
+            for (let i = 0; i < match_codefence.length; i++) {
+                codefenceSplitted[i] = match_codefence[i].split('\n');
+            }
+
+            // split the textToCheck in lines
+            const textSplitted = textToCheck.split('\n');
+
+            for (let i = 0; i < textSplitted.length; i++) {
+                isCodeFence = false;
```

Comment:
Can you move the let for this command here instead? It doesn't need to be declared outside here.

Reply:
yes